.github/workflows/ubuntu.yml

```diff
@@ -146,7 +146,10 @@ jobs:
 
   coverage:
     name: Coverage and Warnings
-    runs-on: ubuntu-latest
+# Since the latest Ubuntu image, lcov fails complaining about negative branch counts,
+# and using fprofile-update=atomic as suggested does not help, so use the previous image
+#    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
complain = error? I think this should be fixed on the gcos-gc3 branch - and not with a downgrade

Reply:
See [Issue 212](https://github.com/OCamlPro/gnucobol/issues/212). I haven't been able to fix that, so I went with a downgrade so that I could continue focusing on the merge.